api/controllers/roomController.js

```diff
@@ -36,10 +36,19 @@ const times = catchAsync(async (req, res) => {
   return res.status(200).json({ data: times });
 });
 
+const join = catchAsync(async (req, res) => {
+  const userId = req.user.id;
```

Comment:
이미 user 정보가 있으므로 여기서 id가 아닌 user 정보를 전달한다면, service에서 다시 user를 호출하지 않아도 될 것 같습니다.
```suggestion
  const user = req.user;
```